mite/context.py

```diff
@@ -99,9 +122,17 @@ async def transaction(self, name):
                 e.handled = True
                 raise
         finally:
-            self.send('txn', start_time=start_time, end_time=time.time(), had_error=error)
+            self.send(
+                'txn',
+                start_time=start_time,
+                end_time=time.time(),
+                had_error=error,
+                context=path,
+                meta=get_transaction_meta()
```

Comment:
And also paying for extra data in the pipeline for every txn (even if it's just an empty dict by default)? 😕